### PR TITLE
Feature/bla 2 project setup

### PR DESCRIPTION
**Pull request by nablanco**
Created new react app project, deleted unnecessary files, and add dependencies (ESLint, Prettier, Chakra UI, and Styled Components)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  "name": "bookshelf_app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@chakra-ui/react": "^2.8.2",
+    "@emotion/react": "^11.13.0",
+    "@emotion/styled": "^11.13.0",
+    "@testing-library/jest-dom": "^5.17.0",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
```
Comment (ThomasEmilsson) [inline package.json]: Can we move these to be `devDependencies` ?

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
```
Comment (ThomasEmilsson) [inline public/index.html]: Update these 😄

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (ThomasEmilsson) [inline public/index.html]: Same here 😄

File: src/App.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+
+
+function App() {
+  return (
+    <div className="App">
+    </div>
```
Comment (ThomasEmilsson) [inline src/App.js:6]:
```suggestion
    <div className="App"/>
```

File: public/manifest.json [hunk 5]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```
Comment (ThomasEmilsson) [inline public/manifest.json]: Update these too 🦞

File: package.json (hunk 1 above)
Comment (ThomasEmilsson) [reply package.json]: Also `eslint` and `prettier`

**Review by ThomasEmilsson — CHANGES_REQUESTED** [review]
Looks good just a couple of minor changes/suggestions

File: package.json (hunk 1 above)
Comment (nablanco) [reply package.json]: Remove eslint and prettier?

Comment (ThomasEmilsson) [reply package.json]: Can move them to `devDependencies` I mean

Comment (nablanco) [reply package.json]: thanks!

**Review by ThomasEmilsson — APPROVED** [review]

**Pull request merged**